README.md

```diff
@@ -200,6 +200,58 @@ pipenv shell
 apt install lintian
 make test
 ```
+## Signing sources
+
+SecureDrop Workstation code spans across the following repositories:
+
+* https://github.com/freedomofpress/securedrop-client
+* https://github.com/freedomofpress/securedrop-debian-packaging
+* https://github.com/freedomofpress/securedrop-proxy
+* https://github.com/freedomofpress/securedrop-sdk
+* https://github.com/freedomofpress/securedrop-workstation
+* https://github.com/freedomofpress/qubes-template-securedrop-workstation
+
+
+### Release
+1. For each release, a tag for each release will be signed and pushed to each of the above repos.
+
+2. Create a Makefile target in securedrop-debian-packaging repo that contains release tags / commit hashes for each repository used for the release. To verify the tag signature and check out the packaging logic:
+```
+git tag -v <tag>
+git checkout <tag>
+```
+
+3. Metadata (e.g. commit hash for release) should be tracked inside the .deb (e.g.: `/usr/share/packagename/release-info.txt`)
```

Comment:
After giving more thoughts on this during holiday, I think the better place can be the changelog of the debian package itself. As that will make sure the final things getting installed on the system are only from the tarball, and nothing dynamic there. We can add a check to verify this during final packaging CI (when we will have it).